### PR TITLE
Add Permission Checks to PrisonerLocationView

### PR DESCRIPTION
**Pull request by marcofucci**
This makes sure that only users beloging to the PrisonerLocationAdmin group can access the PrisonerLocation API.

### CODE REVIEW

**Comment by marcofucci** [comment]
@SteveMarshall @ibrechin review/merge pls?

File: mtp_api/apps/core/tests/utils.py [hunk 1]
```diff
@@ -3,16 +3,21 @@
...
 from prison.models import Prison
-from mtp_auth.tests.mommy_recipes import create_prison_user_mapping
+from mtp_auth.tests.mommy_recipes import create_prison_user_mapping, \
+    create_prisoner_location_admins
 
 
 def make_test_users(users_per_prison=1):
-    users = []
+    # prison clerks
+    prison_clerks = []
```
Comment (SteveMarshall) [inline mtp_api/apps/core/tests/utils.py]: Not sure this needed a comment :wink:

**Comment by SteveMarshall** [comment]
:+1:

@ibrechin?

File: mtp_api/apps/core/tests/utils.py (hunk 1 above)
Comment (marcofucci) [inline mtp_api/apps/core/tests/utils.py]: It's not a comment, it's a h2 kind of thing ;-)

**Comment by ibrechin** [comment]
Looks good.

**Pull request merged**